src/v2/view-builder/views/TerminalView.js

```diff
@@ -4,7 +4,7 @@ import { getBackToSignInLink, getSkipSetupLink, getReloadPageButtonLink } from '
 import EmailAuthenticatorHeader from '../components/EmailAuthenticatorHeader';
 import { OTPInformationTerminalView } from './consent/EmailMagicLinkOTPTerminalView';
 import CustomAccessDeniedErrorMessage from './shared/CustomAccessDeniedErrorMessage';
-import EndUserRemediationTerminalMessage from './end-user-remediation/EndUserRemediationTerminalMessage';
+import EndUserRemediationMessages from './end-user-remediation/EndUserRemediationMessages';
```

Comment:
Renamed to remove the "terminal" language since this template is now being used by grace period and the terminal end user remediation flow